deploy/release.groovy

```diff
@@ -2,15 +2,13 @@
 def ci (){
     stage('build planner npm'){
         container('ui'){
-            sh 'npm install'
-            sh 'npm run build'
-            sh 'npm pack dist/'
+            sh 'npm run build-planner'
```

Comment:
You haven't changed anything but making it more opaque. I do NOT think this is good.